frontend/src/component/common/Search/SearchSuggestions/SearchSuggestions.tsx

```diff
@@ -69,16 +71,19 @@ export const SearchSuggestions: VFC<SearchSuggestionsProps> = ({
                 getColumnValues(column, row)
             );
 
+            const options = [...new Set(filterOptions)]
+                .filter(Boolean)
+                .flatMap(item => item.split('\n'))
+                .filter(item => !item.includes('"') && !item.includes("'"))
```

Comment:
if your field value is using quotes (corner case) we filter it out to not mess with the search quotes